setup.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
 
-# Start ignoring PyImportSortBear as imports below may yield syntax errors
+# Start ignoring PyImportSortBear and PycodestyleBear* as imports below may
+# yield syntax errors
 from bears import assert_supported_version
 assert_supported_version()
```

Comment:
@Abhisek- you're right. If we keep it like this in case of an older version of python the code won't break. It will only show a warning.